display.py

```diff
@@ -11,9 +11,11 @@
 
 import logging
 
-##If you have a Waveshare v2 screen, try changing this to 
-#from waveshare_epd import epd7in5_V2 as epd7in5
-from waveshare_epd import epd7in5 as epd7in5
+if os.getenv("WAVESHARE_EPD75_VERSION",1):
+    from waveshare_epd import epd7in5 as epd7in5
+else:
+    from waveshare_epd import epd7in5_V2 as epd7in5
```

Comment:
Based on the var, the right Python module should be imported

Reply:
I may suggest setting the default to "2", as that is the model that is actively being sold on Amazon right now for newbies like me.

Reply:
Yes good point. I will change it now.